Exercise_1.py

```diff
@@ -0,0 +1,51 @@
+# Problem1
+# Search a 2D Matrix(https://leetcode.com/problems/search-a-2d-matrix/)
```

Comment:
You have named the questions wrongly.  This solution is for searching in rotated sorted array